notification-portlet-api/src/main/java/org/jasig/portlet/notice/NotificationAction.java

```diff
@@ -50,6 +50,7 @@
     private String id = getClass().getSimpleName();
     private String label;
     private String apiUrl;
+    private boolean redirect = false;
```

Comment:
Would this be a good place for https://projectlombok.org/features/GetterSetter ?

Reply:
I refactored and tested with @Data.